po/it.po

```diff
@@ -2243,39 +2258,44 @@ msgstr "Copia _URL"
 # pynicotine/gtkgui/downloads.py
 # pynicotine/gtkgui/search.py
 # pynicotine/gtkgui/uploads.py
-#: pynicotine/gtkgui/search.py:421 pynicotine/gtkgui/search.py:1132
+#: pynicotine/gtkgui/search.py:422 pynicotine/gtkgui/search.py:1156
 msgid "Copy Folder U_RL"
 msgstr "Copia U_RL Cartella"
 
 # pynicotine/gtkgui/chatrooms.py
-#: pynicotine/gtkgui/search.py:423 pynicotine/gtkgui/search.py:1135
+#: pynicotine/gtkgui/search.py:424 pynicotine/gtkgui/search.py:1159
 #: pynicotine/gtkgui/transferlist.py:197 pynicotine/gtkgui/transferlist.py:756
 msgid "User(s)"
 msgstr "Utente(i)"
 
 # pynicotine/gtkgui/about.py
 # pynicotine/gtkgui/nicotine_glade.py
-#: pynicotine/gtkgui/search.py:428
+#: pynicotine/gtkgui/search.py:429
 msgid "Copy Search Term"
 msgstr "Copia Termine di Ricerca"
 
-#: pynicotine/gtkgui/search.py:430
+#: pynicotine/gtkgui/search.py:431
 msgid "Clear All Results"
 msgstr "Pulisci Tutti i Risultati"
 
-#: pynicotine/gtkgui/search.py:570
+#: pynicotine/gtkgui/search.py:571
 msgid "Filtered out excluded search result "
 msgstr "Risultati di ricerca esclusi dal filtro "
 
-#: pynicotine/gtkgui/search.py:575
+#: pynicotine/gtkgui/search.py:576
 msgid "Filtered out inexact or incorrect search result "
 msgstr "Risultati di ricerca inesatti o incorretti esclusi dal filtro "
 
-#: pynicotine/gtkgui/search.py:925 pynicotine/gtkgui/search.py:927
-msgid "Result Filters"
-msgstr "Filtri Risultato"
+#: pynicotine/gtkgui/search.py:926
+#, python-format
+msgid "Result _Filters [%d]"
```

Comment:
Key accelerator shortcut will be changed to "_Result Filters" see https://github.com/nicotine-plus/nicotine-plus/issues/1500#issuecomment-911158480 sorry. s.